src/App.tsx

```diff
@@ -1,14 +1,26 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [isLoad, setIsLoad] = useState(true);
+  const [isModalSee, setIsModalSee] = useState(false);
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [todo, setTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setIsLoad(false);
+    }, 1000);
+  }, [isLoad]);
```

Comment:
The dependency array for the `useEffect` hook includes `isLoad`. This means the effect will run every time `isLoad` changes, which might not be the intended behavior. If you want the effect to run only once when the component mounts, consider using an empty dependency array `[]`.